Update README.md - add description (#16)
status-im/sticker-market: README.md

# Status.im Sticker Market Dapp

Sticker Market is composed of 3 contracts: 

> - An authorship NFT (StickerType) which is granted to every registered StickerPack in StickerMarket, intended use is to change the deposit address for all StickerPacks sold in StickerMarket.

Reviewer: ```suggestion
- An authorship NFT (`StickerType`) which is granted to every registered `StickerPack` in `StickerMarket`, intended use is to change the deposit address for all `StickerPack`s sold in `StickerMarket`.
```

> - StickerMarket main contract, which is the minter of two other NFT smart contract addresses, StickerType and StickerPack. It allows users to mint StickerType, which contains a content-hash link, and other users to mint StickerPack referenced to certain StickerTypes. 

Reviewer: ```suggestion
- `StickerMarket` main contract, which is the minter of two other NFT smart contract addresses, `StickerType` and `StickerPack`. It allows users to mint `StickerType`, which contains a content-hash link, and other users to mint `StickerPack` referenced to certain `StickerType`s. 
```

> - A purchase NFT (StickerPack) referencing a StickerType, which is granted to any account who paid the value defined in the StickerType while it sale was available. 

Reviewer: ```suggestion
- A purchase NFT (`StickerPack`) referencing a `StickerType`, which is granted to any account who paid the value defined in the `StickerType` while it sale was available. 
```

> Therefore, it models an envoirment where artists can publish their packs for sale, that can be used in official Status app, by holders of StickerPack.  Artists can also sell their token, which new owner could set a new payment address.

Reviewer: ```suggestion
Therefore, it models an environment where artists can publish their packs for sale, that can be used in the official Status app, by holders of `StickerPack`.  Artists can also sell their tokens, which new owners could set a new payment address.
```
